pkg/kubeinteraction/status/task_status_test.go

```diff
@@ -9,10 +9,11 @@ import (
 	testclient "github.com/openshift-pipelines/pipelines-as-code/pkg/test/clients"
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/test/kubernetestint"
 	tektontest "github.com/openshift-pipelines/pipelines-as-code/pkg/test/tekton"
+	"github.com/stretchr/testify/assert"
```

Comment:
i try generally to avoid using testify but yeah if there is no way in gotestyourself then we can do this...

Reply:
there is no in built function i think